packages/block-library/src/group/edit.js

```diff
@@ -28,7 +32,7 @@ function GroupEdit( { hasInnerBlocks, className } ) {
 			{ InspectorControlsColorPanel }
 			<BackgroundColor>
 				<TextColor>
-					<div className={ className } ref={ ref }>
+					<Block.div className={ className } ref={ ref }>
 						<div className="wp-block-group__inner-container">
```

Comment:
Later it would be great to see if/how we can remove this div and instead rely on a more declarative way for InnerBlocks wide/full/left/right/center alignments support.